src/components/AiChat/types.ts

```diff
@@ -1,6 +1,7 @@
 // Some of these are based on (compatible, but simplified / restricted) versions of @ai-sdk/react types.
 
 import { RefAttributes } from "react"
+import { type MathJax3Config } from "better-react-mathjax"
```

Comment:
_I mention this because I think you might find it interesting. Something like this isn't worth worring about IMo unless we enforce it with a linting rule._

```suggestion
import type { MathJax3Config } from "better-react-mathjax"
```
is mildly preferable I think.  If a file looks like this

```ts

const sideEffects = () => console.log('Hello')
sideEffects()

export type Foo = { a: string }
```

then `import { type Foo } ...` runs the side effects, but `import type { Foo }` won't.

I.e., `import type {} ... ` is absolutely guaranteed to have no effect on the final bundle.